testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/naming/remote/ejb/RemoteNamingEjbTestCase.java

```diff
@@ -264,4 +282,17 @@ public void testSystemPropertyConfig() throws Exception {
             Thread.currentThread().setContextClassLoader(current);
         }
     }
+
+    @Test
+    public void testValue() throws Exception {
+        ModelNode address = getAddress();
+        ModelNode operation = getOperation(address);
+        ModelNode res = executeOperation(operation);
```

Comment:
In the end this is a call to AbstractMgmtTestBase.executeOperation(operation, true) which is just a call to ManagementOperations.executeOperation(getModelControllerClient(), operation). So this is just ManagementOperations.executeOperation(managementClient.getControllerClient(), operation), which can be called directly.